src/drawers/canvasDrawer.ts

```diff
@@ -20,6 +20,9 @@ export type CanvasDrawStep = (
  * This class is immutable (but has internal state) and shouldn't be extended.
  */
 export class CanvasDrawer implements IDrawer {
+  protected _lastDrawnData: any[];
+  protected _lastProjector: AttributeToAppliedProjector;
```

Comment:
this is the simplest way to allow the drawer to report the bounds of the element. this is necessary for entityNearest to work with canvas drawers